lib/account_deleter.rb

```diff
@@ -69,10 +68,6 @@ def delete_standard_person_associations
     end
   end
 
-  def disconnect_contacts
```

Comment:
See commit-message: Also remove `disconnect_contacts` methods, because contacts are already removed with aspects memberships in `before_destroy`.

I had a problem after changing this to use `person` instead of `diaspora_handle`, because it tried to do this twice, but the second time it failed, because `contact.user` was nil.